vit/command_bar.py

```diff
@@ -73,19 +63,23 @@ def set_edit_text(self, text, edit_pos=None):
 
     def set_command_prompt(self, caption, edit_text=None):
         self.set_caption(caption)
-        if edit_text:
+        if edit_text is not None:
             self.set_edit_text(edit_text)
 
     def activate(self, caption, metadata, edit_text=None):
         self.set_metadata(metadata)
         self.set_command_prompt(caption, edit_text)
 
-    def cleanup(self, command):
-        self.set_caption('')
-        self.set_edit_text('')
-        self.history.cleanup(command)
+    def deactivate(self):
```

Comment:
Hm, this still doesn't feel quite right to me. The purpose of the method is to perform all actions necessary to reset the command bar to a 'fresh' state. How about something like ```reset_data``` or ```reset_configuration```?

Reply:
The name 'deactivate' was proposed becaused 1) the function undoes `activate()`, and 2) it signifies that calling the function turns the command bar into an unusable state, which is not apparent from 'reset' or 'cleanup'.

You have the final say.

Reply:
You talked me into it, let's leave ```deactivate``` as the method name, at least for now.